### ISSUE TITLE
[AccessibilityInsight] Luminosity contrast ratio of the border lines of the Column Header&Cell are less than 3:1 when disabled HeaderVisualStyles for DataGridView

### ISSUE THREAD

**Issue by Lydia-Shi** [dotnet/winforms#6000]
* .NET Core Version: 
.NET 6.0.100-rtm.21513.37

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**
Luminosity contrast ratio of the border lines of Column Header and cell are less than 3:1.
Scenario1:
![image](https://user-images.githubusercontent.com/54019172/137707680-78184655-0e19-4e49-a37d-65aef40ed1ce.png)
Scenario2:
![image](https://user-images.githubusercontent.com/54019172/137707704-52d9e522-c1fd-4d89-9d70-a8f3f322958e.png)

**Expected behavior:**
Luminosity contrast ratio of the border lines of Column Header and cell are greater than 3:1.

**More info:**
Scenario1 just occurs on win11 OS when setting High contract to Deset, cannot repro on Win10.
Scenario2 just occurs on win10 OS when setting High contract to HC white, cannot repro on win11.

**Minimal repro:**
**Scenario1:**
1.	Setting High contract to Desert in win11 os
2.	Create a Winfroms .NET Core application with a DataGridView control
3.	Setting its EnableHeadersVisualStyles to false
4.	Build and run application
5.	Observe the border line of DataGridVew Column Header

**Scenario2:**
1. Set the High Contrast to HC White.
2.  Create a WinForms .NET Core application with a DataGridView control.
3. Set its EnableHeadersVisualStyles is False and RightToLeft to Yes.
4. Build and run application.
5. Observe the border line of DataGridViewCell.

**Comment by vladimir-krestov** [comment]
@merriemcgaw, should we care about contrasts if `EnableHeadersVisualStyles` is turned off?

**Comment by merriemcgaw** [comment]
I'm going to say no. The default is with visual styles on and that is accessible. I think we can close this issue (but good find!)
